caproto/tests/conftest.py

```diff
@@ -211,7 +214,10 @@ def stop_repeater():
         return
 
     logger.info('[Repeater] Sending Ctrl-C to the repeater')
-    _repeater_process.send_signal(signal.SIGINT)
+    if sys.platform == 'win32':
+        _repeater_process.terminate()
```

Comment:
I think we can simplify by using this method on both platforms. On POSIX it [sends SIGTERM](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.terminate) which is just as good.

Reply:
SIGINT (Ctrl-C) and SIGTERM are different. Perhaps we should be adding SIGTERM handlers to our long-running processes to make them clean up and exit gracefully?

Reply:
Until I just checked, I knew they were different but didn't realize that SIGTERM did not have a default handler in Python to convert it to `KeyboardInterrupt`. I'll open a separate issue about adding SIGTERM handlers, and we'll leave this more expedient solution in place for this PR.